compiler/damlc/tests/daml-test-files/LFNameCollisions.daml

```diff
@@ -15,6 +15,8 @@ submit x _ _ = x
 submitMustFail : a -> Int -> Int -> a
 submitMustFail x _ _ = x
 
+-- @WARN Reference to Symbol kind
+-- @WARN Reference to Symbol kind
 data Phantom (a : Symbol) (b : Symbol) = Phantom {}
 
 unpackPair : forall a b c d. Phantom a b -> (c,d) -> (c,d)
```

Comment:
For what we're testing here, this should be enough, isn't it?
```suggestion
unpackPair : forall (a: Symbol) (b: Symbol) c d. (c,d) -> (c,d)
```
If so, what do you think about changing it this way, either here or in a follow-up PR, in order to not conflate this test with out `Symbol` tests?

Reply:
You're right! I changed it, thanks :-)